framework/behaviors/AttributeBehavior.php

```diff
@@ -138,4 +147,25 @@ protected function getValue($event)
 
         return $this->value;
     }
+
+    /**
+     * Returns the value for the current attribute.
+     * This method is called by [[evaluateAttributes()]]. Its return value will be assigned
+     * to the attribute corresponding to the triggering event.
+     * @param Event $event the event that triggers the current attribute updating.
+     * @param string $attribute the attribute name.
+     * @return mixed the attribute value
+     */
+    protected function getValueByAttribute($event, $attribute)
+    {
+        if (array_key_exists($attribute, $this->values)) {
```

Comment:
So... `null` is correct value for `value` (`value` is null in current representation by default)